recipes-libraries/deepview-rt/deepview-rt_2.4.42-aarch64.bb

```diff
@@ -0,0 +1,69 @@
+DESCRIPTION = "This package includes the updated and experimental ModelRunner for TensorFlow Lite and ARM NN. Also in this repository is a pre-release of DeepViewRT with support for the OpenVX backend."
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = "file://COPYING;md5=d3c315c6eaa43e07d8c130dc3a04a011"
```

Comment:
@otavio actually this md5 is not valid for latest EULA (not in fsl-eula-unpack.bbclass). Any plans to add 5.15.5-2.0.0 license?

Reply:
https://github.com/Freescale/meta-freescale/pull/1149